app/controllers/main_controller.rb

```diff
@@ -1,4 +1,16 @@
 class MainController < ApplicationController
   def index
+    @card = Card.expired.random
+  end
+  def check
```

Comment:
Assignment Branch Condition size for check is too high. [20.12/15]<br>Use empty lines between method definitions.